content/projects/gsoc/2021/project-ideas/jenkins-custom-distribution-service-updates.adoc

```diff
@@ -0,0 +1,74 @@
+---
+layout: gsocprojectidea
+title: "Custom Jenkins distribution service improvements"
+goal: "Improve the out of the box solution for packaging Jenkins distributions as WAR files or Docker images"
+category: Tools
+year: 2020
+status: published
+skills:
+- Java
+- Web Development
+- Docker
+mentors:
+#To check on the channel post reviews
+#- "martinda"
+#- "sladyn98"
+- "kwhetstone"
+links:
+  gitter: "jenkinsci/platform-sig"
+---
+
+Many users will download Jenkins and spend a great deal of time customizing their environment to be exactly what they need.
+Often this is done through trial and error.
+It would not only be nice to create common configurations to share with team memebers, but with the wider community.
+
+Last summer, a project (link:https://github.com/jenkinsci/custom-distribution-service[custom-distribution-service]) created the base application with a UI and support for uploading configurations.
+There were a lot of suggestions over the course of the summer, to the point where it would make sense to continue the work over another summer.
+The project is currently hibernating though would be useful after issues are addressed.
+
+The service was temporarily offered as a website at \https://customize.jenkins.io.
```

Comment:
It is still offered I guess the UI is still there but I am not too sure about the backend service